### PR TITLE
Bugfix/colour by category url logic 400 error

### PR DESCRIPTION
**Pull request by lingyun1010**
I fix the logic in the url fetching stage, which is more straight forward, as the `colourByCategory` is decided by the vlaue of `colourBy`

### CODE REVIEW

File: packages/scxa-tsne-plot/src/TSnePlotView.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ class TSnePlotView extends React.Component {
 
   _fetchAndSetStateCellClusters(
     {atlasUrl, experimentAccession, accessKey, selectedColourBy, selectedColourByCategory, selectedPlotType, selectedPlotOptionLabel}) {
-    const resource = selectedColourByCategory === `clusters` ?
+    const resource = !isNaN(selectedColourBy) ?
```
Comment (ke4) [inline packages/scxa-tsne-plot/src/TSnePlotView.js:60]: Is there a test for this code change? Thanks

**Review by ke4 — CHANGES_REQUESTED** [review]
I am missing a test for the code change.
Could you please add it or please describe me why we don't need it?
Thanks

**Review by upendrakumbham — APPROVED** [review]
ALGTM!

**Comment by lingyun1010** [comment]
> I am missing a test for the code change. Could you please add it or please describe me why we don't need it? Thanks

I have looked into this issue. The true or false logic just change the value of a variable not affect any other props or state or UI staff, so we don't have a test for this.

**Pull request merged**